### DISCUSSION TITLE
The encrypted DNS checkmark does not work

### DISCUSSION THREAD

**Discussion by kazantsev97** [AdguardTeam/AdGuardHome#7286] [Q&A]
![Screenshot_2024-09-25-05-51-32-062_com android chrome~2](https://github.com/user-attachments/assets/205146dd-4c74-477d-a4a2-58cfabe21117)
The encrypted dns checkmark does not work, if I disable unencrypted dns, then the Internet immediately stops working, how do I make the encrypted dns work? there is a domain name

**Comment by schzhn** [comment]
Hello, @kazantsev97. You need to configure your device to use encrypted DNS instead of plain DNS. For example `192.168.1.1` -> `https://hostname/dns-query`. Check `Setup Guide` page (`Инструкция по настройке`) in the Admin Web Interface.
